README.rst

```diff
@@ -159,10 +159,6 @@ tooling to provide:
   the full label information of both the sender and receiver among a lot of
   other information.
 
-- Policy decision tracing: Why is a packet being dropped or a request rejected.
-  The policy tracing framework allows to trace the policy decision process for
-  both, running workloads and based on arbitrary label definitions.
```

Comment:
I wonder if we should mention https://app.networkpolicy.io here instead?

Reply:
So far I didn't bother, since this README section is describing the functionality of the Cilium project. I can see arguments that maybe we should point to it somewhere here, but it wasn't entirely obvious to me how or where it makes sense to link it.

Reply:
^^ Open to proposals here but I don't plan on addressing it as part of this feature deprecation PR.

Reply:
Makes sense, that was mostly just me thinking out loud anyway. OK let's keep it at the current version.